docs/tasks.md

```diff
@@ -149,3 +149,12 @@ Updates CMR metadata to set access constraints
 
 - Schemas: See this module's [schema definitions](https://github.com/nasa/cumulus/tree/master/tasks/update-cmr-access-constraints#readme/schemas).
 - Resources: [npm](https://npmjs.com/package/@cumulus/update-cmr-access-constraints) | [source](https://github.com/nasa/cumulus) | [web](https://github.com/nasa/cumulus/tree/master/tasks/update-cmr-access-constraints#readme)
+
+---
+
+### [@cumulus/update-granules-cmr-metadata-file-links](https://github.com/nasa/cumulus/tree/master/tasks/update-granules-cmr-metadata-file-links)
+
+Update cmr metadata files with correct online access urls and etags and transfer etag info to granules' CMR files
```

Comment:
Can we capitalize `CMR`?

Reply:
We can, however this is a direct merge of already approved PRs from master->feature branch.    I can put a PR in against master re: this later.